src/components/notification/notification.ts

```diff
@@ -68,10 +68,9 @@ export class SbbNotificationElement extends LitElement {
   @property({ reflect: true, type: Boolean }) public readonly = false;
```

Comment:
for the notification examples I am not fully happy that pressing "add notification" doesn't have the animation activated. would it be possible to have it simliar to the earlier implementation?